scripts/gen_manifests.sh

```diff
@@ -50,6 +45,11 @@ SONATAFLOW_CR=manifests/01-sonataflow_${workflow_id}.yaml
 yq --inplace eval '.metadata.annotations["sonataflow.org/profile"] = "prod"' "${SONATAFLOW_CR}"
 
 yq --inplace ".spec.podTemplate.container.image=\"quay.io/orchestrator/serverless-workflow-${workflow_id}:latest\"" "${SONATAFLOW_CR}"
+yq --inplace ".spec.podTemplate.container.envFrom=[{\"secretRef\": { \"name\": \"notifications-secret\"}}]" "${SONATAFLOW_CR}"
```

Comment:
perhaps the blame is on me here, but the `-secret` suffix seems redundant here.
let's change it to `notifications-token` which better describes it.
In addition, let's also expose a flag "MOUNT_NOTIFICATIONS_SECRET" enabled by default.
....
few seconds later:
What if we'll have to address additional secrets? perhaps the name should be generalized 
so based on our "friend", the following can be good candidates for the secret name:
- workflow-credentials
- workflow-tokens

Reply:
In that case I think we should go with 1 secret per workflow, like it's done with the CM `-props`

And we can even reuse that naming for the secret: `{{ workflow.id }}-secret-props` (or `obfuscated-props` if you do not like having the word `secret`) WDYT?

Reply:
And no one is to blame, it's a common effort and shared ideas to get something working the best way :)

Reply:
+1 for the suggestion.

Me and @rgolangh where toying with this idea and even thought to contribute this functionality to `kn` cli, so in addition to generating configmaps, it will generate also a secret (perhaps could be fed from a secret.properties) and be patched into the workflow CR.
So let's go with secret per workflow, named `{{ workflow.id }}-creds`

Reply:
Done, I also updated the PR for the doc